.storybook/webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (storybookBaseConfig, configType, defaultConfig) => {
+    defaultConfig.module.rules.push({
+                test: /\.scss$/,
+                use: [
+                    'vue-style-loader',
+                    'css-loader',
+                    'sass-loader'
+                ]
+    });
+    return defaultConfig;
+};
```

Comment:
You need to setup your code editor / IDE to support https://editorconfig.org/

We require to have a new line at the end of each file and it's automagically handled by editors, same as few other things.